docker-compose.yml

```diff
@@ -30,59 +36,95 @@ services:
     image: quay.io/hmpps/hmpps-nomis-prisoner-api:latest
     networks:
       - hmpps
-    container_name: hmpps-nomis-prisoner-api
+    container_name: hpfnm-hmpps-nomis-prisoner-api
     ports:
       - "8082:8080"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - API_BASE_URL_OAUTH=http://hpfnm-hmpps-auth:8080/auth
 
   hmpps-visits-api:
     # For now use migration as a fake visits API
     image: quay.io/hmpps/hmpps-prisoner-to-nomis-update:latest
     networks:
       - hmpps
-    container_name: hmpps-visits-api
+    container_name: hpfnm-hmpps-visits-api
     ports:
       - "8083:8080"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - HMPPS_SQS_LOCALSTACK_URL=http://hpfnm-localstack:4566
+      - API_BASE_URL_OAUTH=http://hpfnm-hmpps-auth:8080/auth
+      - API_BASE_URL_NOMIS=http://hpfnm-hmpps-nomis-prisoner-api:8080
+      - API_BASE_URL_VISITS=http://hpfnm-hmpps-visits-api:8080
+      - API_BASE_URL_VISIT_MAPPING=http://hpfnm-hmpps-nomis-visit-mapping-api:8080
 
 
   hmpps-nomis-visit-mapping-api:
     image: quay.io/hmpps/hmpps-nomis-visits-mapping-service:latest
     networks:
       - hmpps
-    container_name: hmpps-nomis-visit-mapping-api
+    container_name: hpfnm-hmpps-nomis-visit-mapping-api
     ports:
       - "8084:8080"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - SPRING_FLYWAY_URL=jdbc:postgresql://hpfnm-mapping-db:5432/mapping?sslmode=prefer
+      - SPRING_R2DBC_URL=r2dbc:postgresql://hpfnm-mapping-db:5432/mapping?sslmode=prefer
+      - API_BASE_URL_OAUTH=http://hpfnm-hmpps-auth:8080/auth
 
   localstack:
     image: localstack/localstack:0.14.0
     networks:
       - hmpps
-    container_name: localstack
+    container_name: hpfnm-localstack
+    environment:
+      - SERVICES=sqs,sns
+      - DEBUG=${DEBUG- }
+      - DOCKER_HOST=unix:///var/run/docker.sock
+      - DEFAULT_REGION=eu-west-2
+      - HOSTNAME_EXTERNAL=hpfnm-localstack
+    volumes:
+      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
+      - "/var/run/docker.sock:/var/run/docker.sock"
+
+  localstack-external:
```

Comment:
needs to be in twice and configured for external access for one and internal access for the other